Header header
int8 height
int8 width
int8[] shingles
int8[] depot_positions
InchwormMsg[] inchworms
int8[] inchworm_occ